src/platform/site-wide/component-library-analytics-setup.js

```diff
@@ -275,20 +275,55 @@ const analyticsEvents = {
       action: 'change',
       event: 'int-checkbox-option-click',
       prefix: 'checkbox',
+      ga4: {
+        event: 'interaction',
+        component_name: 'va-checkbox',
+        custom_string_1: 'component-library',
+        mapping: {
+          'checkbox-label': 'heading_1',
+          'checkbox-description': 'heading_2',
```

Comment:
I don't think these are headings (label and description)

```
'checkbox-label': 'heading_1',
'checkbox-description': 'heading_2',
```

Reply:
I chose `heading_1` for `checkbox-label`, as labels usually function as a kind of header for the component they belong to. However, I can see the argument for using a `custom_string` instead.

For `checkbox-description`, I agree that the use of `heading_2` isn't an optimal choice; `custom_string_2` would likely be a more suitable option. The challenge here is that GA4 only provides a single `custom_string` event for mapping (namely, `custom_string_2`). The `custom_string_1` variable is essentially a constant that will always have the value `component-library`, so it's off the table. From what I understand too, I can't use GA4 events that aren't listed in the document, so `custom_string_3` was also off the table.

Given these constraints, when considering the possible events in the GA4 planning document, none seem to perfectly fit a label or a description. As such, I decided to use `heading_1` and `heading_2` to try and maintain consistency, but I'm definitely open to hearing suggestions on other ways to map this!

Reply:
> From what I understand too, I can't use GA4 events that aren't listed in the document, so custom_string_3 was also off the table.

I guess we just need to clarify that with Michelle. I have run into this problem before too and I don't think I got a clear answer: https://dsva.slack.com/archives/CKCRMLGAC/p1684515650855819?thread_ts=1683833527.536999&cid=CKCRMLGAC